src/query/storages/fuse/src/operations/gc.rs

```diff
@@ -508,6 +588,16 @@ impl FuseTable {
                 .await?;
         }
 
+        let inverted_index_count = inverted_indexes_to_be_purged.len();
+        if inverted_index_count > 0 {
+            counter.inverted_indexes += inverted_index_count;
+            self.try_purge_location_files_and_cache::<InvertedIndexDirectory, _, _>(
```

Comment:
after PR #15362 merged, inverted index will be readed as `InvertedIndexMeta` and multiple `InvertedIndexFile`s, including:
1. fast file
2. store file
3. fieldnorm file
4. position file
5. idx file
6. term file
7. meta.json file
8. .managed.json file

The cache key is a combination of a file path and a subfile name,
for example, if the inverted index file path is `1/23819/_i_i/idx1/307a34e/cf1ecaf64ff541bb8a2919df86e73d80_v0.index`
the cache key of each subfile is 
`23819/_i_i/idx1/307a34e/cf1ecaf64ff541bb8a2919df86e73d80_v0.index-fast`
`23819/_i_i/idx1/307a34e/cf1ecaf64ff541bb8a2919df86e73d80_v0.index-store`
...

Reply:
got it, thanks

Reply:
@b41sh  please take another look